src/pylon_camera/pylon_camera_parameter.cpp

```diff
@@ -45,7 +45,7 @@ PylonCameraParameter::PylonCameraParameter() :
         binning_y_given_(false),
         downsampling_factor_exp_search_(1),
```

Comment:
Yep, this should be removed as part of moving everything to dynamic reconfigure.